libs/designsystem/src/lib/components/avatar/avatar.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="avatar" [ngClass]="{ shadow: shadow, overlay: overlay }">
+<div class="avatar" [ngClass]="{ shadow: shadow, overlay: overlay, stroke: stroke }">
```

Comment:
You could also apply the `stroke` class if `shadow` is true, and then remove `shadow` from this class binding. Remember to remove the class from the styles as well.

Reply:
Updated!